### PR TITLE
Bot Pull-Request Refactoring with 'sonarqube'

### PR DESCRIPTION
**Pull request by Samantha-Bo**
Hi, I'm a refactoring bot. I found and fixed some code smells for you. 

 You can instruct me to perform changes on this pull request by creating line specific (review) comments inside the 'Files changed' tab of this pull request. Use the english language to give me instructions and do not forget to tag me (using @) inside the comment to let me know that you are talking to me.

### CODE REVIEW

File: src/main/java/smells/incorrectorderofmodifiers/IncorrectOrderOfModifiers.java [hunk 1]
```diff
@@ -10,7 +10,7 @@
 	/**
 	 * expected: private static final int
 	 */
-	private final static int MULTIPLIER = 10;
+	private static final int MULTIPLIER = 10;
```
Comment (MarvinWyrich) [inline src/main/java/smells/incorrectorderofmodifiers/IncorrectOrderOfModifiers.java:13]: @Samantha-Bo Reorder modifiers!

Comment (Samantha-Bo) [reply src/main/java/smells/incorrectorderofmodifiers/IncorrectOrderOfModifiers.java:13]: Could not exchange data with wit.ai! Please rephrase your instruction in a new comment.

Comment (MarvinWyrich) [reply src/main/java/smells/incorrectorderofmodifiers/IncorrectOrderOfModifiers.java:13]: @Samantha-Bo please reorder modifiers :)

Comment (Samantha-Bo) [reply src/main/java/smells/incorrectorderofmodifiers/IncorrectOrderOfModifiers.java:13]: All modifiers are in correct order! Nothing to refactor. Please rephrase your instruction in a new comment.

**Pull request closed without merging**